main.go

```diff
@@ -49,6 +54,21 @@ func (l *logWriter) Write(b []byte) (int, error) {
 	return len(b), nil
 }
 
+type serialPort struct {
+	port string
+}
+
+func (s *serialPort) Write(b []byte) (int, error) {
+	c := &serial.Config{Name: s.port, Baud: 115200}
+	p, err := serial.OpenPort(c)
+	if err != nil {
+		return 0, err
+	}
+	defer p.Close()
+
+	return p.Write(b)
+}
```

Comment:
Should this logic be in `main.go` or would a new `logger.go` module make more sense?

Reply:
Separate files in the same package just act as logical separation, this was such a small addition and main.go is not overloaded I thought an extra file was overkill (boilerplate would be longer than actual code)